spec/requests/comments_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+require './spec/support/helpers'
+
+RSpec.describe "Comments", type: :request do
+  describe "GET /index" do
+    let!(:user) {create(:user)}
+    let(:worker) {user.worker}
+    let!(:ticket) {create(:ticket)}
+    let!(:comment) {create(:comment)}
+    let(:params) { {
+      "user": {
+        "email": "hello@mail.com",
+        "password": "secret",
+        "password_confirmation": "secret",
+        "worker_attributes": {
+          "first_name": "dffdfdfdfdfd",
+          "last_name": "Bradi",
+          "age": 30,
+          "role": "Developer" } } }
+    }
+
+    before do
+      post '/users', params: params
+    end
+
+    it "index return a valid http status 200 " do
+      get'/tickets/1/comments'
+      expect(response).to have_http_status(200)
+    end
+
+    it "index return a valid http status 200" do
+      get'/tickets/333/comments'
```

Comment:
you may move `/tickets/333/comments` into let(:url) 
or event 
let(:get_request) { get '/tickets/333/comments' }

but don't use hardcoded ids